settings.gradle.kts

```diff
@@ -68,6 +68,9 @@ dependencyResolutionManagement {
             version("com.google.auto.service", "1.1.1")
             version("org.hyperledger.besu.nativelib.secp256k1", "0.8.2")
             version("info.picocli", "4.7.6")
+            version("com.github.luben.zstd_jni", "1.5.6-8")
+            version("com.google.cloud.core", "2.48.0")
+            version("com.google.cloud.storage", "2.45.0")
```

Comment:
```suggestion
```

AS you have the versions in `build.gradle.kts` directly, you do not need them here.

Note: with #216, we can move these versions together witth the others to `hiero-dependency-versions/build.grlad.kts`. I will do that then.

Reply:
Note, `version("com.github.luben.zstd_jni", "1.5.6-8")` is being used elsewhere as well, it should remain here I think.